src/context/WalletProvider/WalletProvider.test.tsx

```diff
@@ -27,6 +27,10 @@ jest.mock('@shapeshiftoss/hdwallet-metamask', () => ({
   },
 }))
 
+jest.mock('p-debounce', () => ({
+  promise: jest.fn(fn => fn),
+}))
+
```

Comment:
See https://github.com/sindresorhus/p-debounce/issues/18 - needed or jest will crash since this is a pure ESM package

Reply:
Update: moved to a global mock not to repeat the mock every time we consume `p-debounce.promise` in a module (i.e when we consume `useIsSnapInstalled()` / `checkIsSnapInstalled()`